lldb/source/Plugins/Process/elf-core/ThreadElfCore.h

```diff
@@ -183,6 +203,7 @@ class ThreadElfCore : public lldb_private::Thread {
 
   int m_signo;
   int m_code;
+  std::string m_sig_description;
 
```

Comment:
See `lldb::StopInfoSP` comment above. We seem to be duplicating info from the `siginto_t` struct in both the `ThreadData` and in the `ThreadELFCore` classes.